src/main/resources/templates.conf

```diff
@@ -24,6 +24,7 @@ templatecontrol {
     play-scala-fileupload-example,
     play-scala-forms-example,
     play-scala-isolated-slick-example,
+    play-scala-log4j2-example,
```

Comment:
This example project was not being maintained. It did have branches for 2.5.x and 2.6.x and the master was outdated. See https://github.com/playframework/play-scala-log4j2-example/pull/2, https://github.com/playframework/play-scala-log4j2-example/pull/3, and https://github.com/playframework/play-scala-log4j2-example/pull/4 for details.